internal/impl/ockam/input_ockam_kafka_franz.go

```diff
@@ -0,0 +1,193 @@
+package ockam
+
+import (
+	"context"
+	"math/rand"
+	"net"
+	"strconv"
+	"strings"
+
+	"github.com/benthosdev/benthos/v4/internal/impl/kafka"
+	"github.com/benthosdev/benthos/v4/public/service"
+)
+
+func ockamFranzKafkaInputConfig() *service.ConfigSpec {
+	return service.NewConfigSpec().
```

Comment:
maybe we could just call the base implementation for this (rather than copy-paste the fields) and add the ockam' related field to it?.  something like  
```
return kafka.FranzKafkaInputConfig().
                Summary(`An Ockam wrapper around Kafka input using the [Franz Kafka client library](https://github.com/twmb/franz-go).`).
		Field(service.NewStringField("node_name").Optional()).
		Field(service.NewStringField("node_port").Optional()).
		Field(service.NewStringField("inlet_address").Optional())
```
 (I'm assuming Summary() overrides the previous Summary() by the underling plugin).
This way there is no need to keep both in sync.

Reply:
Absolutely, thanks!